grid/booking/list-activities/maps/list-activities.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jest/no-export */
+import { SuperfaceTest } from '@superfaceai/testing';
+
+import { buildSuperfaceTest } from '../../../test-config';
+
+export function listActivitiesTest(provider: string) {
+  describe(`booking/list-activities/${provider}`, () => {
+    let superface: SuperfaceTest;
+
+    beforeEach(() => {
+      superface = buildSuperfaceTest({
+        profile: 'booking/list-activities',
+        provider,
+      });
+    });
+
+    describe('ListActivities', () => {
+      it('should perform successfully', async () => {
```

Comment:
:+1: 
What pattern would you suggest? Adding the pagination logic to the one test that is there?

I'd like it to only run if `nextPage` is returned, seeing as some future providers (or even Bokun) may only return a number of activities smaller than the page, so I tried...
1. conditional tests (i.e. `if (result.nextPage) { it('should paginate'), ... }`), but that's a jest lint error.
2. to make it a test that's conditionally *skipped*, but then I got an error because that's a nested test
3. Make it a completely independent test, but for that, I need the pagination token - but I don't want to make `perform`s (in order to retrieve it) outside of test logic...
4. Make it a completely independent test that queries the first page (again) - but once I'm in the test, I can't skip it, and just having it pass feels wrong (the fact that the test didn't run should be clearly signalled, as it may be an error)

Reply:
I'd go with option 4, I think I did something similar for social-media/posts with LinkedIn. In case you aren't able to test the pagination with the given provider, you can easily separate that test out.